src/NBomber/DomainServices/ScenarioRunner.fs

```diff
@@ -35,43 +35,42 @@ let run (config: Contracts.Scenario, isVerbose: bool) =
     let scenario = Scenario.create(config)
     Log.Information("{Scenario} has started", config.ScenarioName)        
 
-    let result = initScenario(scenario)
-                 |> Result.bind(warmUpScenario)
-                 |> Result.map(startFlows)
-    match result with
-    | Ok actorsHosts ->
-        Log.Information("wait {time} until the execution ends", scenario.Duration.ToString())
-        Log.Information("processing...")
+    let result = initScenario(scenario) |> Result.bind(warmUpScenario)
```

Comment:
can you create one function like prepareScenario(scenario): Scenario
and then move all logic (initScenario, warmUp etc) there to get:
scenario |> prepareScenario |> startFlows